src/lib/__tests__/supabase.test.ts

```
@@ -0,0 +1,89 @@
+      update: jest.fn(() => ({
+        eq: jest.fn(() => ({
+          select: jest.fn(() => ({
+            single: jest.fn(),
+          })),
+        })),
+      })),
+      delete: jest.fn(() => ({
+        eq: jest.fn(),
+      })),
+      order: jest.fn(() => ({
+        then: jest.fn(),
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)